components/ide/code/gitpod-web-extension/leeway.Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+FROM node:18 as builder
+
+ARG CODE_EXTENSION_COMMIT
+
+RUN apt update -y \
+    && apt install jq --no-install-recommends -y
+
+RUN mkdir /gitpod-code-web \
+    && cd /gitpod-code-web \
+    && git init \
+    && git remote add origin https://github.com/gitpod-io/gitpod-code \
+    && git fetch origin $CODE_EXTENSION_COMMIT --depth=1 \
+    && git reset --hard FETCH_HEAD
+WORKDIR /gitpod-code-web
+RUN yarn --frozen-lockfile --network-timeout 180000
+
+# update package.json
+RUN setSegmentKey="setpath([\"segmentKey\"]; \"untrusted-dummy-key\")" && \
```

Comment:
@iQQBot @mustard-mh did we check whether analytics still works?

Reply:
checked https://github.com/gitpod-io/gitpod/pull/19652#issuecomment-2079930681